src/Token.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Token.h"
+
+std::string Tolkien::Token::toString()
+{
+    return (TokenTypeString(this->type) + " " + this->lexeme + " " + this->literal);
+}
```

Comment:
what's the story here?

Reply:
I'm following the book steps and doesn't explain (so far) what is the purpose of it, but i believe this will be useful for outputting errors maybe...